sc/social/like/browser/viewlets.py

```diff
@@ -87,14 +87,8 @@ def render(self):
         return ''
 
     def enabled(self):
-        """Check if the viewlet should be shown in this context."""
-        template = self.helper.view_template_id()
-        # If using folder_full_view or all_content, we add metadata
-        # in order to proper display share buttons for
-        # contained content types
-        if template in ('all_content', 'folder_full_view',):
-            return True
-        return self.helper.enabled(self.view)
+        """Enable metadata viewlet globally"""
+        return True
```

Comment:
why are you doing this?

Reply:
these metadata should be used in other places where social like is not necessary.  Other plugins can use those metadatas.

Reply:
The tile likes we did in the new portal for example, we don't need social likes into the Cover type, but need the metadata to add these sharing options.